modules/tinymce/changelog.txt

```diff
@@ -1,4 +1,5 @@
 Version 5.6.0 (TBD)
+    Added new `image_file_types` setting to determine which image file formats will be automatically processed into `img` tags #TINY-6306
```

Comment:
```suggestion
    Added new `image_file_types` setting to determine which image file formats will be automatically processed into `img` tags on paste #TINY-6306
```
Maybe? I just think we need to clarify it only applies when pasting image URL like strings.